dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -5,49 +5,48 @@
 #NLS_MESSAGEFORMAT_NONE
 
 authFilter=Authentication Filter
-authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers to determine whether or not the HTTP request is selected for the authentication.
+authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers. If all conditions are met, the HTTP request is selected for the authentication.
 
 webApp=Web Application
-webApp.desc=Specifies the web application name.
+webApp.desc=Specifies the web application name. For example, <webApp id="sample" name="app1|app2" matchType="notContain"/>
```

Comment:
Suggested update:
"Specifies the web application name, for example, `<webApp id="sample" name="app1|app2" matchType="notContain"/>`."